operator/cmd/run/run.go

```diff
@@ -501,6 +528,47 @@ func Run(
 		}
 	}
 
+	if enableStatefulSetDecommissioner {
```

Comment:
nit: If you find yourself doing any other touch ups it'd be nice to pull this out into it's own function so we can reduce the total length of Run.

Reply:
i looked at this but did not find it useful without further refactoring, i will need to pass stuff like the setupLog etc.